### DISCUSSION TITLE
I cannot figure how to have Typeahead working.

### DISCUSSION THREAD

**Discussion by ItsLemmy** [abenz1267/walker#144] [Q&A] [answered]
The wiki is super vague about the Typeahead feature.
From the config it seems that it should be available for Runner and Emojis.

I have "typeahead": true for both modules.  But I have never seen the typeahead feature in action. 
Am I missing something ? I have double checked my CSS to no avail.

Thanks in advance for the help!

**Comment by abenz1267** [comment]
Pls share your config, i'll have a look if something is wrong.

**Reply by ItsLemmy** [reply]
[walker-rosepine.tar.gz](https://github.com/user-attachments/files/17599724/walker-rosepine.tar.gz)

**Comment by abenz1267** [comment]
![image](https://github.com/user-attachments/assets/1a351d93-52c2-4889-a886-8b397ba8ce95)

This is with your config... works fine?

The typeahead gives you a hint you can autocomplete with `<tab>`. In that case i only typed in `shu` so far. Pressing <tab> would autocomplete to `shut now` and put the last item i activated with that on top.

**Comment by ItsLemmy** [comment]
Very weird, I get this...  Does it require any external dependencies I would be missing on my system?

![image](https://github.com/user-attachments/assets/fff461a5-ef93-4b84-a0a7-f1db53a295a3)

**Reply by abenz1267** [reply]
Stupid question: but did you actually run an entry?

**Reply by abenz1267** [reply]
Doesnt require anything, no.

**Reply by ItsLemmy** [reply]
Not a stupid question, that was it!

Any reason this is only available for Runner and Emojis ?

**Reply by abenz1267** [reply, marked as answer]
It's available for every module. Runner and emoji just have it enabled by default. By default, most modules don't really need it.

**Reply by ItsLemmy** [reply]
Ok gotcha!
Thanks, closing this.
